qa-bloodpac.planx-pla.net/manifest.json

```diff
@@ -4,88 +4,123 @@
+    "pidgin": "quay.io/cdis/pidgin:2021.03",
+    "revproxy": "quay.io/cdis/nginx:2021.03",
+    "sheepdog": "quay.io/cdis/sheepdog:2021.03",
+    "portal": "quay.io/cdis/data-portal:2021.03",
     "fluentd": "fluent/fluentd-kubernetes-daemonset:v1.2-debian-cloudwatch",
     "jupyterhub": "quay.io/occ_data/jupyterhub:2020.05",
-    "spark": "quay.io/cdis/gen3-spark:2020.08",
-    "tube": "quay.io/cdis/tube:2020.08",
-    "ssjdispatcher": "quay.io/cdis/ssjdispatcher:2020.08",
-    "guppy": "quay.io/cdis/guppy:2020.08",
-    "hatchery": "quay.io/cdis/hatchery:2020.08",
+    "spark": "quay.io/cdis/gen3-spark:2021.03",
+    "tube": "quay.io/cdis/tube:2021.03",
+    "ssjdispatcher": "quay.io/cdis/ssjdispatcher:2021.03",
+    "guppy": "quay.io/cdis/guppy:2021.03",
+    "hatchery": "quay.io/cdis/hatchery:2021.03",
     "ambassador": "quay.io/datawire/ambassador:1.4.2",
-    "wts": "quay.io/cdis/workspace-token-service:2020.08",
-    "manifestservice": "quay.io/cdis/manifestservice:2020.08"
+    "wts": "quay.io/cdis/workspace-token-service:2021.03",
+    "manifestservice": "quay.io/cdis/manifestservice:2021.03",
+    "dashboard": "quay.io/cdis/gen3-statics:2021.03"
+  },
+  "indexd": {
+    "arborist": "true"
+  },
+  "ssjdispatcher": {
+    "job_images": {
+      "indexing": "quay.io/cdis/indexs3client:2021.03"
+    }
   },
   "arborist": {
     "deployment_version": "2"
   },
+  "sower": [
+    {
+      "name": "manifest-indexing",
+      "action": "index-object-manifest",
+      "activeDeadlineSeconds": 86400,
+      "serviceAccountName": "jobs-qa-bloodpac-planx-pla-net",
+      "container": {
+        "name": "job-task",
+        "image": "quay.io/cdis/gen3/manifest-indexing:2021.02",
+        "pull_policy": "Always",
+        "env": [
+          {
+            "name": "GEN3_HOSTNAME",
+            "valueFrom": {
+              "configMapKeyRef": {
+                "name": "manifest-global",
+                "key": "hostname"
+              }
+            }
+          }
+        ],
+        "volumeMounts": [
+          {
+            "name": "sower-jobs-creds-volume",
+            "readOnly": true,
+            "mountPath": "/creds.json",
+            "subPath": "creds.json"
+          }
+        ],
+        "cpu-limit": "1",
+        "memory-limit": "1Gi"
+      },
+      "volumes": [
+        {
+          "name": "sower-jobs-creds-volume",
+          "secret": {
+            "secretName": "sower-jobs-g3auto"
+          }
+        }
+      ],
+      "restart_policy": "Never"
+    }
+  ],
   "jupyterhub": {
     "enabled": "yes",
     "nb_storage_size": "250Gi",
-    "sidecar": "quay.io/cdis/gen3fuse-sidecar:2020.08",
+    "sidecar": "quay.io/cdis/gen3fuse-sidecar:2021.03",
     "containers": [
       {
         "name": "Bioinfo - Python/R",
         "cpu": 0.5,
         "memory": "256M",
-        "image": "quay.io/occ_data/jupyternotebook:1.7.4",
-        "env": {
-          "NAMESPACE": "qa-bloodpac"
```

Comment:
also these blocks should be kept

I think we might not need this `jupyterhub` section because we have individual hatchery config json, but just for sake of keeping bloodpac qa and prod in sync 🤷‍♂️